### ISSUE TITLE
[Accessibility] Research and Compile Plan to Improve Accessibility

### ISSUE THREAD

**Issue by zakuArbor** [zakuArbor/blog#14]
It's been in my mind for a few weeks on the fact I should be caring more about accessibility. Some ideas are to research W3 standards on accessibilities and MathJax accessibility extension. This is not expected to be done for a very long time and probably will not fix older blog posts unless requested.
